packages/web/pages/players.tsx

```diff
@@ -15,7 +15,10 @@ import React, { useEffect, useMemo, useRef } from 'react';
 
 type Props = InferGetStaticPropsType<typeof getStaticProps>;
 
+
+
 export const getStaticProps = async () => {
+  console.log("WE MADE IT")
```

Comment:
Oops :D 

No changes of significance, let's just revert back to the version in development